app/models/zip_endpoint.rb

```diff
@@ -65,6 +65,10 @@ def audit_class
     raise "Failed to return audit class based on setting for #{endpoint_name}.  Check setting string for accuracy."
   end
 
+  def to_s
+    endpoint_name
```

Comment:
Add test coverage? Or CodeClimate may be mistaken.

Reply:
i think codeclimate was right, added a test that specifically exercised this via the `AuditResults` `storage_area` generalization that prompted it, and diff coverage went up to 100%.